### DISCUSSION TITLE
Deployer v8 Alpha

### DISCUSSION THREAD

**Discussion by antonmedv** [deployphp/deployer#3973] [General]
[Deployer v8 Alpha](https://deployer.org/blog/deployer-v8-alpha)

**Comment by mrmaarek** [comment]
![Scherm­afbeelding 2024-11-29 om 14 24 14](https://github.com/user-attachments/assets/14ee475c-39d2-40ea-a6ac-516e026e7f50)

I would expect `noThrow` instead of `nothrow` though :)

**Reply by antonmedv** [reply]
This decision was influenced by my other open-source project zx: https://google.github.io/zx/process-promise#nothrow
:)
